src/engine/renderer/tr_image.cpp

```diff
@@ -145,13 +145,19 @@ void R_ListImages_f()
 
 	typedef std::pair<const std::string, int> nameSizePair;
 	static const std::unordered_map<uint32_t, nameSizePair> imageFormatNameSize = {
-		// TODO: find imageDataSize multiplier
+		/* Generic format that does not have any specified actual format
+		   and implementations can choose one of a few different ones
```

Comment:
Better avoid any space between  the indentation and the comment text.

The only places where I have seen somewhat meaningful sub-comment indentation was for code quotes or things like that (a bit like a kind of in-comment-markdown).

Reply:
I prefer it left-aligned, but sure I'll make it aligned to the /* itself.

Reply:
Changed it to align to /*.